apps/charterafrica/src/payload/utils/nestCollectionUnderPage.js

```diff
@@ -16,7 +16,7 @@ function nestCollectionUnderPage(pageSlug) {
     } catch (error) {
       // TODO(kilemensi): Add Sentry to payload & report errors
     }
-    return { ...doc, link: { href } };
+    return { ...doc, docLink: doc.link ?? null, link: { href } };
```

Comment:
What has changed here?

Reply:
Here there's a field called link in tools which is overwritten by this link in afterread hook. So I'm adding this to ensure we don't lose that data

Reply:
Again, wasn't this supposed to be handled when importing from Airtable?


> **Airtable + GitHub -> CMS**: Do all necessary pre-processing and standardisation
> **CMS -> Website**: Localisation/formatting for display only.


What am I missing?

Reply:
You're right! My bad.

Reply:
I'm assuming we've tested this and all is good. If yes, please resolve the conversation.